pkg/webhook/server/webhook_cert_init.go

```diff
@@ -0,0 +1,177 @@
+//
+// Copyright (c) 2019-2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+package server
+
+import (
+	"context"
+
+	"github.com/che-incubator/che-workspace-operator/internal/controller"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func InitWebhookServer(ctx context.Context) error {
+	crclient, err := controller.CreateClient()
+	if err != nil {
+		return err
+	}
+
+	log.Info("Creating the secure service")
+	secureService := createSecureService()
+	err = crclient.Create(ctx, secureService)
+	if err != nil {
+		return nil
+	}
+
+	log.Info("Creating the secure configmap")
+	createSecureConfigMap := createSecureConfigMap()
+	err = crclient.Create(ctx, createSecureConfigMap)
+	if err != nil {
+		return nil
+	}
+
+	log.Info("Updating the deployment")
+	err = updateDeployment(ctx, crclient)
+	if err != nil {
+		return nil
+	}
+
+	return nil
+}
+
+func createSecureService() *v1.Service {
```

Comment:
To avoid confusion, I would avoid naming this with `create` (since it could imply that the service is created on cluster). Maybe something like `getSecureServiceSpec()`? 

Similar suggestion for `createSecureConfigMap`